.github/workflows/update-mds.yml

```
@@ -0,0 +1,45 @@
+name: Update README.md
+
+on:
+  pull_request_target:
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          token: ${{secrets.CONTANE_BOT_TOKEN}}
+      - name: Configure Git
+        run: |
+          git config user.name "contane-bot"
+          git config user.email "160241315+contane-bot@users.noreply.github.com"
+      - name: Install readme-generator-for-helm
+        run: npm install -g @bitnami/readme-generator-for-helm
```

We should really pin this to prevent updates from breaking our workflow unexpectedly.

```suggestion
        run: npm install -g @bitnami/readme-generator-for-helm@2.6.1
```

... and we would need to update this via Renovate, I'm afraid.